development/core-updates/9.0/bo-rebrand.md

```diff
@@ -0,0 +1,27 @@
+---
+title: BO rebranding
```

Comment:
You can put this content inside the Notable Changes page that already exists:
https://devdocs.prestashop-project.org/9/modules/core-updates/9.0/

You can create a new section, "Back office rebranding", at the end of this page.